.github/workflows/helm-chart-lint.yaml

```diff
@@ -41,20 +51,22 @@ jobs:
         run: |
           changed=$(ct list-changed --target-branch ${{ github.event.repository.default_branch }})
           if [[ -n "$changed" ]]; then
-            echo "CHART_CHANGED=true" >> $GITHUB_ENV
+            echo "changed=true" >> $GITHUB_OUTPUT
           fi
 
-      # run chart linting
       - name: Run chart-testing (lint)
-        run: ct lint --target-branch ${{ github.event.repository.default_branch }} --config charts/config/chart-testing-config.yaml
```

Comment:
did you verified the chart-testing-config.yaml with @nicoprow if there is a reason you can also let this in.

also for `--validate-maintainers=false`

Reply:
Yea right, we should include         the parameter --config charts/config/chart-testing-config.yaml again